skia/RSkSVGTextContainer.cpp

```diff
@@ -57,11 +61,22 @@ void  RSkSVGTextContainer::updateCommonTextProps(SharedProps newViewProps) {
 
   SET_TEXT_FRAME_ATTR(SkSVGAttribute::kX,newRNSVGTextPropsProps.x,0)
   SET_TEXT_FRAME_ATTR(SkSVGAttribute::kY,newRNSVGTextPropsProps.y,0)
-
-#ifdef ENABLE_EXPERIMENTAL_DELTA_ATTRIBUTE_SUPPORT
   SET_TEXT_FRAME_ATTR(static_cast<SkSVGAttribute>(RSkSVGAttribute::kDX),newRNSVGTextPropsProps.dx,0)
   SET_TEXT_FRAME_ATTR(static_cast<SkSVGAttribute>(RSkSVGAttribute::kDY),newRNSVGTextPropsProps.dy,0)
-#endif/*ENABLE_EXPERIMENTAL_DELTA_ATTRIBUTE_SUPPORT*/
+
+  if((newRNSVGTextPropsProps.x.size() > 1 ) || (newRNSVGTextPropsProps.y.size() > 1 )  ||
+     (newRNSVGTextPropsProps.dx.size() > 1 ) || (newRNSVGTextPropsProps.dx.size() > 1 ) ){
+    RNS_LOG_WARN("Positioning each character in a content is not supported");
```

Comment:
use NOT_IMPL macro. Multiple positioning ( x and delta) not supported

Reply:
modified to RNS_LOG_NOT_IMPL_MSG